src/main/scala/sbtavro/DefaultSchemaParserBuilder.scala

```diff
@@ -0,0 +1,34 @@
+package sbtavro
+
+import java.util
+
+import com.spotify.avro.mojo.SchemaParserBuilder
+import org.apache.avro.Schema
+
+case class DefaultSchemaParserBuilder(types: util.Map[String, Schema],
```

Comment:
Nice ! we should probably move `types` to a scala collection and call java converters in the build. This will give a cleaner api in sbt.

Reply:
I think that makes sense. Also I will make it List instead, as key is just schema name (and not used anyways later in Avro implementation for this call :D)